src/scss/_packages.scss

```diff
@@ -2,80 +2,172 @@
 @import "mixins";
 
 .r-tiles {
-    @apply items-center;
-
     @screen md {
-        @apply flex;
+        @apply grid grid-cols-2 grid-flow-row-dense gap-8;
```

Comment:
Why grid here, and not just flex-wrap?

Reply:
From my discussion with @susanev these cards needed to grow independently of their sibling cards. Honestly this is actually incomplete still and I think what we want to do is a Masonry-like layout but that is not easily achievable, in my opinion, with either a Grid or a Flexbox layout. I might be wrong, though? I wanted to get this in and come back to looking at the layout of cards specifically in a follow-up PR. If you prefer to switch these to use a Flexbox layout for now, that's fine.

I'll talk to you about this separately with some additional details.